kodata/rotations/1-serving.rotation

```diff
@@ -0,0 +1,23 @@
+# Serving community contacts
+#
+# Begin metadata
+#@ title: Serving
+#@ slack: #serving-questions
+#@ slacklink: https://knative.slack.com/archives/C0186KU7STW
+
+2020-12-14T17:00:00Z | tcnghia
```

Comment:
I'm okay with other formats, but this was pretty easy to write a parser for and is is pretty easy to read and grok as a human.

Reply:
I couldn't find a good package for this. Unfortunately, it seems like most of the systems for this like PagerDuty don't have configuration files, they have APIs that aren't really very human-config-file manageable.

I'm modelling this off escalator configs, for reference. 😉